crates/ruma-common/src/push/condition.rs

```diff
@@ -168,10 +170,24 @@ impl PushCondition {
                 }
                 RoomVersionFeature::_Custom(_) => false,
             },
+            Self::_Custom(_) => false,
         }
     }
 }
 
+/// An unknown push condition.
+#[doc(hidden)]
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[allow(clippy::exhaustive_structs)]
+pub struct _CustomPushCondition {
+    /// The kind of the condition.
+    pub kind: String,
+
+    /// The additional fields that the condition contains.
+    #[serde(flatten)]
+    pub data: BTreeMap<String, JsonValue>,
```

Comment:
Why are these fields `pub`? If they need to be accessible from another module, `pub(super)` or `pub(crate)` should be sufficient, right?

Reply:
Just the habit of declaring fields `pub` :sweat_smile: